### PR TITLE
Devex - Entity constants refactoring

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: web-client/src/views/EditDocketEntry/PrimaryDocumentForm.jsx [hunk 1]
```diff
@@ -268,13 +270,14 @@ export const PrimaryDocumentForm = connect(
             validateSequence={validateDocketEntrySequence}
           />
 
+          {console.log(addDocketEntryHelper.showObjection, '****')}
```
Comment (rachaelparris) [inline web-client/src/views/EditDocketEntry/PrimaryDocumentForm.jsx]: Remove?

File: web-client/src/views/AddDocketEntry/PrimaryDocumentForm.jsx [hunk 2]
```diff
@@ -308,7 +310,7 @@ export const PrimaryDocumentForm = connect(
                 <legend className="usa-legend" id="objections-legend">
                   Are there any objections to the granting of this document?
                 </legend>
-                {['Yes', 'No', 'Unknown'].map(option => (
+                {OBJECTIONS_OPTIONS.map(option => (
```
Comment (rachaelparris) [inline web-client/src/views/AddDocketEntry/PrimaryDocumentForm.jsx:313]: It looks like the options in EntityConstants are in a different order - could you update to match the old ordering?

Comment (rachelschneiderman) [reply web-client/src/views/AddDocketEntry/PrimaryDocumentForm.jsx:313]: Yes! I realized I didn't do 'Yes' or 'No' either. I will push the order fix with those updates as well.

**Review by kkoskelin — APPROVED** [review]
once comments are addressed, 👍

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/getConstants.js [hunk 3]
```diff
@@ -126,6 +128,8 @@ export const getConstants = () =>
     MAX_FILE_SIZE_BYTES,
     MAX_FILE_SIZE_MB,
     NOTICE_EVENT_CODES: DOCUMENT_NOTICE_EVENT_CODES,
+    OBJECTIONS_OPTIONS,
+    OBJECTIONS_OPTIONS_MAP: OBJECTIONS_OPTIONS_MAP,
```
Comment (kkoskelin) [inline web-client/src/getConstants.js:132]: _nit_: Can probably use shorthand notation?

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/getConstants.js (hunk 3 above)
Comment (rachelschneiderman) [reply web-client/src/getConstants.js:132]: Yes you got it, will include in the other PR!

**Pull request merged**